clients/admin-ui/src/features/privacy-requests/configuration/MailgunEmailConfiguration.tsx

```diff
@@ -19,7 +19,7 @@ const MailgunEmailConfiguration = () => {
     useState<ConnectionStep>("");
   const { handleError } = useAPIHelper();
   const { data: messagingDetails } = useGetMessagingConfigurationDetailsQuery({
-    type: "mailgun",
+    type: "MAILGUN",
```

Comment:
Should this be a string literal constant declared in the file or an enum configuration type located in a constants.ts file?